protos/node_reputation/node_reputation.proto

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+package nodereputation;
+
+service NodeReputation {
+    rpc NodeReputation(NodeQuery) returns (NodeReputationRecord) {}
+    rpc UpdateReputation(NodeUpdate) returns (UpdateReply) {}
+    rpc FilterNodeReputation(NodeFilter) returns (NodeReputationRecords) {}
+    rpc PruneNodeReputation(NodeQuery) returns (UpdateReply) {}
+}
+
+message NodeReputationRecord {
+    string source = 1;
+    string nodeName = 2;
+	string timestamp = 3;
+	int64 uptime = 4;
+	int64 auditSuccess = 5;
+	int64 auditFail = 6;
+	int64 latency = 7;
+	int64 amountOfDataStored = 8;
+	int64 falseClaims = 9;
+    int64 shardsModified = 10;
+    float score = 11;
```

Comment:
as i'm sure you've gathered, i'm anxious about score, haha. i'd rather not have an answer than an incomplete one. we really need to think through the incentives here, and the sip9 approach didn't work